framework/src/main/java/org/checkerframework/common/aliasing/AliasingTransfer.java

```diff
@@ -111,12 +133,14 @@ protected void processPostconditions(
         }
 
         // Now, doing the same as above for the receiver parameter
```

Comment:
Inner class constructors can also have receivers. So shouldn't there also be a check? Potentially, this is related to #282

Reply:
Can you add a todo with a reference to that issue and comment on that issue?

Reply:
Thanks!